scripts/archive_docs.py

```diff
@@ -138,9 +138,23 @@ def recursive_copy(source, target, version):
             recursive_copy(source_path, target_path, version)
 
 
+def archive_single_pdf_document(version):
+    copy_file(
+        'single-file-document/duckdb-docs.pdf',
```

Comment:
So whoever runs the archive script will have to download the file or generate it?